src/quo/components/community/community_detail_token_gating/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo.components.community.community-detail-token-gating.view
+  (:require [clojure.string :as string]
+            [quo.components.community.community-detail-token-gating.style :as style]
+            [quo.components.dividers.divider-label.view :as divider-label]
+            [quo.components.markdown.text :as text]
+            [quo.components.tags.collectible-tag.view :as collectible-tag]
+            [quo.components.tags.token-tag.view :as token-tag]
+            [react-native.core :as rn]
+            [utils.i18n :as i18n]))
+
+(defn- token-view
+  [{:keys [collectible? img-src amount sufficient?] :as token}]
+  (if collectible?
+    [collectible-tag/view
+     {:collectible-name    (:symbol token)
```

Comment:
`symbol` can be destructured in line 12

Reply:
Destructing `symbol` directly will result in `Shadowed var: cljs.core/symbol` lint error.